recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,27 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
-    movie_ratings = None
-    '''
-    complete the code to compute a variable movie_ratings
-    1. perform a merge between data_movies and data_ratings on 'movieId'.
-    2. group by title and find the mean of ratings and fetch top 20 records.
-    3. convert the records to dict.
```

Comment:
It is better to initialize 'response' as an empty list more explicitly. Instead of 'response = []', consider using 'response = list()' for clarity.